pages/establishment/update/routers/confirm.js

```diff
@@ -22,7 +23,8 @@ module.exports = () => {
       }, params)
     };
 
-    return req.api(`/establishment/${req.establishmentId}`, opts);
+    return req.api(`/establishment/${req.establishmentId}`, opts)
+      .then(saveTaskIdToSession(req.session));
```

Comment:
This one should move down to live alongside the other handling code at L67-ish so it is structured consistently with other implementations.

Reply:
It's here because of L32 - we wrap the `sendData()` function in `updateDataFromTask()` which does not return the original response from sendData

Reply:
Basically anywhere we've got a separate `sendData()` function, I've added it there because it's sometimes gets used with additional middleware.